src/liblcvm.cc

```diff
@@ -212,17 +214,22 @@ int get_frame_drop_info(const char *infile, int *num_video_frames,
 
   // 6. calculate frame drop ratio as extra drop length over total duration
   *frame_drop_ratio = drop_length_duration_sec / total_duration_sec;
-
+  *frame_drop_count = drop_length_sec_list_sum / delta_timestamp_sec_median;
```

Comment:
This looks wrong. Why are we dividing to calculate the frame drop count?

Reply:
What about?

    *frame_drop_count = drop_length_sec_list.size()

This may be +1 or -1. Please make sure you test it

Reply:
drop_length_sec_list this has only cluster values, i thought requirement was to get exact number frames dropped.
ex: if there are 2 cluster frame drops but could have 10 frames dropped.

Reply:
delta_timestamp_sec_list.size()

?

Reply:
No, ```delta_timestamp_sec_list``` contains  interframe difference of all the frames not just frame drops or clusters.
this is how we calculate total duration

```python
 float total_duration_sec = 0.0;
  for (const auto &delta_timestamp_sec : delta_timestamp_sec_list) {
    total_duration_sec += delta_timestamp_sec;
  }
```